lambda/iam.tf

```diff
@@ -2,6 +2,10 @@ locals {
   secret_and_kms_policy_json = length(var.secrets_and_kms_arns) > 0 ? data.aws_iam_policy_document.main.json : data.aws_iam_policy_document.dummy.json
 }
 
+data "aws_iam_role" "main" {
+  name = var.lambda_role_name
```

Comment:
does this work, when `var.lambda_role_name`?

if no, we could do sth like:


```
name = var.lambda_role_name == null ? aws_iam_role.main.name : var.lambda_role_name
```

and then you can continue to use `data.aws_iam_role.main` through the rest of the code and don't need to duplicate the ternary operator all the time.
```

Reply:
Great suggestions! 👍